tests/api/utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+from typing import Dict, Iterable, List, Callable
+
+
+def get_messages_by_predicate(
+    messages: Iterable[Dict], predicate: Callable[[Dict], bool]
+) -> List[Dict]:
+    return [msg for msg in messages if predicate(msg)]
```

Comment:
Is this function really useful ?
It introduces the uncommon term `predicate` and only contains a trivial one line of code.

Reply:
I use it a ton in another PR, just thought I'd introduce it here so that it's already in. I can rename 'predicate' to 'cond', that works as well.